app/build.gradle

```diff
@@ -61,6 +61,7 @@ android {
     test {
       java.srcDirs += "src/sharedTest/java"
       kotlin.srcDirs += "src/sharedTest/java"
+      java.exclude('**/DataBinderMapperImpl.java')
```

Comment:
Please add a comment to explain and provide context on why this is being done. Otherwise people might accidentally change it without understanding the nuance that you do.

Reply:
Done! Let me know if it needs a more in-depth explanation here - I figured the documentation in the actual file might provide this.

Reply:
This looks good--thanks!